src/config.ts

```diff
@@ -61,13 +63,15 @@ function loadConfig() {
   const milliseconds = seconds * 1000;
   const memRaw = parsedEnv.data.SANDBOX_MEMORY_LIMIT;
   const cpuRaw = parsedEnv.data.SANDBOX_CPU_LIMIT;
+  const filesDir = parsedEnv.data.FILES_DIR ?? '';
```

Comment:
Given that FILES_DIR is optional and nullable, I believe we can remove the `?? ''` safely, otherwise we might end up with a filesDir which is an empty string. Better to have it undefined if it's not specified by the user.